docs/samples/modelmesh_sdk_sample.ipynb

```diff
@@ -0,0 +1,562 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Sample for ModelMesh Serving using the KServe Python SDK\n",
```

Comment:
I think this should be a more concise title, like ...
- "Python SDK for ModelMesh"
- "Python SDK Examples"
- "How to use the KServe Python SDK with ModelMesh"
-